src/main/java/nextstep/ladder/Line.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.ladder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Line {
+    private final List<Point> points;
+
+    public Line(List<Point> points) {
+        this.points = points;
+    }
+
+    public static Line of(int countOfPerson) {
+        List<Point> points = new ArrayList<>();
+        Point point = new Point();
+        points.add(0, point);
```

Comment:
index 설정은 없어도 될 것 같은데 추가하신 이유가 궁금합니다 🤔 
```suggestion
        points.add(point);
```

Reply:
특별한 이유는 없었습니다!  
불필요한 index 설정 제거하겠습니다!